src/bcrhp/bcrhp/pkg/map_layers/mapbox_spec_json/overlays/streams/streams.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "Streams",
+  "sources": {
+    "stream-networks-source": {
+      "type": "vector",
+      "tiles": [
+        "/int/bcrhp/bctileserver/geo/pub/WHSE_BASEMAPPING.FWA_STREAM_NETWORKS_SP/ows?service=WMS&request=GetMap&layers=WHSE_BASEMAPPING.FWA_STREAM_NETWORKS_SP&styles=&format=pbf&transparent=true&version=1.3.0&height=512&width=512&srs=EPSG:3857&bbox={bbox-epsg-3857}"
+      ],
+      "tileSize": 512
+    }
+  },
+  "arches-metadata": {
+    "ispublic": false,
+    "addtomap": false,
+    "centerx": null,
+    "centery": null,
+    "icon": "fa fa-globe",
+    "isoverlay": true,
+    "legend": "<p><img alt=\"\" class=\"streams legend-line\" src=\"/int/bcrhp/static/img/blank.png\" />Streams</p>",
+    "searchonly": false,
+    "sortorder": 0,
```

Comment:
This is showing up first. Should check with emerald if they still want it and if so where it should be in the list.

Reply:
Thanks for fixing!